source/actors/Dash.gd

```diff
@@ -0,0 +1,20 @@
+extends "res://actors/Walk.gd"
```

Comment:
Not sure if Dash should inherit from Walk, although they really share they basic behavior. I might redesign this later